src/software/world/robot.h

```diff
@@ -132,6 +123,15 @@ class Robot
      */
     std::set<RobotCapability> &getMutableRobotCapabilities();
 
+    /**
+     * Decides if a point is near the dribbler of the robot
+     *
+     * @param test_point The point to check
+     *
+     * @return whether the test_point is near the dribbler of the robot
```

Comment:
In the `robot.c`, we define "near" by if the point is within the 90 degree cone in front of the robot. Would it better if we included this in the javadoc? Could be better than just "near the dribbler".

Reply:
the implementation details shouldn't be in the java doc, but I can add comments to the function body

Reply:
The function body already has a comment on it (I think) so no need.